src/UI/molecules/Performance.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { memo, Fragment } from "react";
+
+import FlexContainer from "<layout>/FlexContainer";
+import PerformanceItem from "<molecules>/PerformanceItem";
+
+import {
+  PerformanceItemProps
+} from "<molecules>/__typings__/PerformanceItem.d.ts";
+
+import getRandomDelay from "<helpers>/math/getRandomDelay";
+
+const performanceItems: PerformanceItemProps[] = [
+  { title: "Total heap size" },
```

Comment:
```suggestion
const performanceItems: PerformanceItemProps[] = [
  "Total heap size",
```
This could be just an array of strings - objects are not necessary